### ISSUE TITLE
Page does not have a main landmark

### ISSUE THREAD

**Issue by kimberlythegeek** [mozilla-services/screenshots#5016]
[detailed info on landmark-one-main rule](https://dequeuniversity.com/rules/axe/3.1/landmark-one-main?application=axeAPI)
```
Rule Violated:
landmark-one-main - Ensures the page has only one main landmark and each iframe in the page has at most one main landmark
	URL: https://dequeuniversity.com/rules/axe/3.1/landmark-one-main?application=axeAPI
	Impact Level: moderate
	Tags: cat.semantics best-practice
	Elements Affected:
	1) Target: html
		Page does not have a main landmark
```

**Comment by kimberlythegeek** [comment]
Also affects region rule
[detailed info on region rule](https://dequeuniversity.com/rules/axe/3.1/region?application=axeAPI)
```
Rule Violated:
region - Ensures all page content is contained by landmarks
	URL: https://dequeuniversity.com/rules/axe/3.1/region?application=axeAPI
	Impact Level: moderate
	Tags: cat.keyboard best-practice
	Elements Affected:
	1) Target: html
		Some page content is not contained by landmarks
```

**Comment by ianb** [comment]
[We are discontinuing the Screenshots server](https://blog.mozilla.org/futurereleases/2019/01/24/clarifying-the-future-of-firefox-screenshots/), and so we are closing this issue.
